src/usr/local/buildpack/tools/python.sh

```diff
@@ -87,23 +87,23 @@ export_tool_path "${USER_HOME}/.local/bin"
 function python_shell_wrapper () {
   local install_dir=$(get_install_dir)
   local FILE="${install_dir}/bin/${1}"
-  check_command ${tool_path}/bin/$1
-  cat > $FILE <<- EOM
+  check_command "${tool_path}"/bin/"$1"
+  cat > "$FILE" <<- EOM
 #!/bin/bash
 
 export PYTHONHOME=${tool_path} PATH=${tool_path}/bin:\$PATH
 
 ${1} "\$@"
 EOM
-  chmod +x $FILE
+  chmod +x "$FILE"
 }
 
-python_shell_wrapper ${TOOL_NAME}
-python_shell_wrapper ${TOOL_NAME}${MAJOR}
-python_shell_wrapper ${TOOL_NAME}${MAJOR}.${MINOR}
+python_shell_wrapper "${TOOL_NAME}"
+python_shell_wrapper "${TOOL_NAME}"${MAJOR}
```

Comment:
seems wrong, `${MAJOR}` should also be quoted

Reply:
```suggestion
python_shell_wrapper "${TOOL_NAME}${MAJOR}"
```